rdr_common/code_constants.py

```diff
@@ -82,3 +82,5 @@
 UNSET = 'UNSET'
 UNMAPPED = 'UNMAPPED'
 BASE_VALUES = [UNSET, UNMAPPED, PMI_SKIP_CODE]
+
+LANGUAGE_OF_CONSENT = ['en', 'es']
```

Comment:
We may want to add a comment saying English and Spanish are the only accepted languages for now.